corehq/apps/hqwebapp/views.py

```diff
@@ -351,6 +351,11 @@ def login(req):
     # this view, and the one below, is overridden because
     # we need to set the base template to use somewhere
     # somewhere that the login page can access it.
+
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
not a bad thought, though I think it's about equivalent, complexity-wise. i'm hoping this code will be the only thing that's strictly for icds, as opposed to the `enterprise_mode` setting.